app/waf/src/repositories/common.ts

```diff
@@ -0,0 +1,28 @@
+import {Entity} from '@loopback/repository';
+import {DataObject, Options, DefaultCrudRepository} from '@loopback/repository';
+import uuid = require('uuid');
+
+export class CommonRepository<
```

Comment:
You may need to extend all repositories from this `CommonRepository` since the respective models are extended from `Common`.

Reply:
Can not do that now, because not all entity are extended from CommonEntity. Need to do that when refactoring other resources.